### PR TITLE
SSL documentation

### PR DESCRIPTION
**Pull request by juliahayward**
Document how to set up SSL support.

### CODE REVIEW

File: documentation/database/mysql.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ subtitle: MySQL
 <td><code>jdbc:mysql://<i>host</i>:<i>port</i>/<i>database</i></code></td>
 </tr>
 <tr>
+<th>SSL support</th>
+<td>Yes - add <code>?useSsl=true</code></td>
+<td>Yes - add <code>?useSsl=true</code></td>
```
Comment (MikielAgutu) [inline documentation/database/mysql.md:34]: Duplicated line

File: documentation/ssl.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+layout: documentation
+menu: ssl
+subtitle: SSL support
+---
+# SSL support
+
+In many scenarios, maintaining a secure connection to the database is important. Flyway can easily be configured to
```
Comment (MikielAgutu) [inline documentation/ssl.md]: I think 'In many scenarios, maintaining a secure connection to the database is important' sort of goes without saying :)

File: documentation/ssl.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+You can also check which certificates have been imported already:
+
+<pre class="console"><span>&gt;</span> keytool -keystore myStorePath -list</pre>
+
+## Make sure Java can access the truststore
+
+You now need to configure your Java environment to be able to access these stores. If you're using the Flyway 
+command-line, then you can simply set the `JAVA_ARGS` environment variable which the command-line script reads.
+
+<pre class="console"><span>&gt;</span> JAVA_ARGS='-Djavax.net.ssl.trustStore="myStorePath" -Djavax.net.ssl.trustStorePassword="myStorePassword"'</pre>
```
Comment (MikielAgutu) [inline documentation/ssl.md:44]: I think it makes sense to link to the blog post about `JAVA_ARGS` https://flywaydb.org/blog/jvm-properties

**Review by MikielAgutu — CHANGES_REQUESTED** [review]

File: documentation/database/mysql.md (hunk 1 above)
Comment (juliahayward) [reply documentation/database/mysql.md:34]: This is intentional, it's putting two columns into a row with the same value in each

File: documentation/ssl.md (hunk 2 above)
Comment (juliahayward) [reply documentation/ssl.md]: I'll reword it a bit, but it's always good to err on the side of caution around how much the readers actually know.

**Review by MikielAgutu — APPROVED** [review]

**Pull request merged**